src/Cassandra/ConsistencyLevel.cs

```diff
@@ -76,6 +76,19 @@ public enum ConsistencyLevel
         /// <summary>
         /// Similar to <c>One</c> but only within the DC the coordinator is in.
         /// </summary>
-        LocalOne = 0x000A
+        LocalOne = 0x000A,
+        YbConsistentPrefix = One,
```

Comment:
Per MS naming convention, 2-letter acronym should be kept in upper case (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers). Rename to `YBConsistentPrefix` and `YBStrong`.